py/desispec/tile_qa_plot.py

```diff
@@ -1291,7 +1291,10 @@ def make_tile_qa_plot(
             n_valid += zhists[sel].sum() * istracer.sum()
 
             if tracer=="QSO" :
-                nqso_rr = int(zhists[sel].sum() * istracer.sum())
+                if istracer.sum() == 0:
+                    log.warning("istracer.sum() == 0 for QSO, probably due to LOW_EFFTIME (TBC); nqso_rr = 0")
```

Comment:
Minor: I suggest that you set `nqso_rr=0` here instead of relying upon that being set before entering the loop over tracers and relying upon this code only being run for tracer="QSO".  I believe the code as-is works, but it potentially fragile to future updates.